### PR TITLE
Re-order next_reminders columns

### PR DESCRIPTION
**Pull request by tomrf1**
the order is different in data-platform:
https://github.com/guardian/data-platform-models/blob/5dd6b0bc8c3689c5c876ec72e12db88eaf753b35/deployment-manager/deployment.jinja#L1714

### CODE REVIEW

**Review by paulbrown1982 — APPROVED** [review]
👍

File: src/next-reminders/lib/db.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-import { Pool, QueryConfig, QueryResult } from 'pg';
-import { runWithLogging } from '../../lib/db';
-import { getCurrentReminderPeriod } from './utils';
+import {Pool, QueryConfig, QueryResult} from 'pg';
+import {runWithLogging} from '../../lib/db';
+import {getCurrentReminderPeriod} from './utils';
```
Comment (TomPretty) [inline src/next-reminders/lib/db.ts:3]: might want to undo this change?

Comment (tomrf1) [reply src/next-reminders/lib/db.ts:3]: I'm not sure where this has come from, but it didn't break the build!

Comment (TomPretty) [reply src/next-reminders/lib/db.ts:3]: Yeah looks like we don't run linting as part of the build. I can create a quick PR to fix it

Comment (TomPretty) [reply src/next-reminders/lib/db.ts:3]: https://github.com/guardian/support-reminders/pull/29
